tools/webpack

```diff
@@ -29,7 +29,18 @@ def run_watch(host, port, minify):
     # type: (str, str, bool) -> None
     """watches and rebuilds on changes, serving files from memory via webpack-dev-server"""
     webpack_args = ['node', 'node_modules/.bin/webpack-dev-server']
-    webpack_args += ['--config', 'tools/webpack.config.ts', '--port', port, "--host", host]
+    webpack_args += [
+        '--config',
+        'tools/webpack.config.ts',
+        '--host',
+        host,
+        '--port',
+        port,
```

Comment:
For readability, I would probably use one line each for pairs of options with a key and value like `host` and `port`.